### PR TITLE
adding comments

### PR DESCRIPTION
**Pull request by sudeshnab19**
Arnab please review and share your feedback

### CODE REVIEW

File: function_tablemultiplier.py
Comment (pogo420) [inline function_tablemultiplier.py:1]: remove these unwanted code.

File: function_tablemultiplier.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def average(a,b):
 """
 
 def table_multiplier(i,x,y):
```
Comment (pogo420) [inline function_tablemultiplier.py:20]:
```suggestion, adding datatype recommendation is good practice.
def table_multiplier(i: int, x: int, y: int):
```
use meaningful name

Comment (pogo420) [reply function_tablemultiplier.py:20]: Instead of print return data from function.

**Review by pogo420 — CHANGES_REQUESTED** [review]
implement these changes.

File: function_tablemultiplier.py
Comment (sudeshnab19) [reply function_tablemultiplier.py:1]: Removed the unwanted code

File: function_tablemultiplier.py (hunk 1 above)
Comment (sudeshnab19) [reply function_tablemultiplier.py:20]: Used meaningful name. 

Will check for return value

**Pull request merged**